### PR TITLE
Ammo rebalance, new antag ammo, sprites. Attempt 2

### PR DESCRIPTION
**Pull request by Voldirs**
Read description and stuff in #2798

### CODE REVIEW

File: code/modules/projectiles/projectile/bullettypes.dm [hunk 1]
```diff
@@ -1,5 +1,12 @@
 /obj/item/projectile/bullet/c9mm
-	damage = 18
+	damage = 22
+	sharp = 0
+	can_ricochet = TRUE
+
+/obj/item/projectile/bullet/c9mmh
+	damage = 1.2*22
```
Comment (NanakoAC) [inline code/modules/projectiles/projectile/bullettypes.dm]: having math in here like this is wierd, best to just precalculate it and write it in.  26.4

Comment (Voldirs) [reply code/modules/projectiles/projectile/bullettypes.dm]: That's for future, so people know how these numbers got calculated. Does it look so weird?

Comment (NanakoAC) [reply code/modules/projectiles/projectile/bullettypes.dm]: yea this is a bad method

If you want to make it based on the base class, then make the HV bullets a subclass, and have them modify the values in their New proc
and also make 1.2 a define

**Comment by NanakoAC** [comment]
Copy of the PM i sent you this morning 

I have an idea on how to setup the ammo code
do it like this:
```
#define HV_MULT 1.2


#define DAMAGE_762 20
#define AP_762 15
/obj/item/projectile/bullet/a762
    damage = DAMAGE_762
    armor_penetration = AP_762
    
/obj/item/projectile/bullet/a762/hv
    damage = DAMAGE_762 * HV_MULT
    armor_penetration = AP_762 * HV_MULT
```
Make defines like that for each ammotype, that way the actual numbers aren't being copypasted around, and its easily maintainable
and right at the bottom of the file make sure to do...
```
#undef HV_MULT
#undef DAMAGE_762
#undef AP_762
#undef everything_else_you_defined```

**Comment by Voldirs** [comment]
>Make the HV bullets a subclass of the main bullet

They have no var shared. Why doing so?

**Pull request merged**